quickwit/Cargo.lock

```diff
@@ -2253,6 +2226,12 @@ dependencies = [
  "libc",
 ]
 
+[[package]]
+name = "hermit-abi"
```

Comment:
do you know what ended up adding this dep?
if possible it would be nice to disable it

Reply:
Ok I checked. This is architecture specific, and we don't pull it.